stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListView.kt

```diff
@@ -276,6 +277,9 @@ public class MessageListView : ConstraintLayout {
 
     private var deletedMessageListItemPredicate: MessageListItemPredicate =
         DeletedMessageListItemPredicate.VisibleToEveryone
+
+    internal var deletedMessageListItemPredicateLiveData = MutableLiveData(deletedMessageListItemPredicate)
```

Comment:
Using `MutableLiveData` because if we were to use simple listeners we would run into the issues of order of initialization in which we have no guarantee that the listener was set before the VM was bound to the View.

By using a stateful, reactive element we are able to guarantee that we are up to date with emissions.